functions.php

```diff
@@ -1,13 +1,23 @@
 <?php
 
-/**
- * Additional code for the child theme goes in here.
- */
-
-add_action( 'wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_enqueue_scripts', 'enqueue_child_styles', 99);
+add_action('wp_head', 'hook_javascript');
 
 function enqueue_child_styles() {
 	$css_creation = filectime(get_stylesheet_directory() . '/style.css');
+	wp_enqueue_style('child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation);
+}
 
-	wp_enqueue_style( 'child-style', get_stylesheet_directory_uri() . '/style.css', [], $css_creation );
+function hook_javascript() {
```

Comment:
Tested locally, it works :+1: 
I would rename this function to something more explicit like `include_fundraiseup_script`, but that works anyway :)